### DISCUSSION TITLE
Android 14 needs to start the service manually and allow to control the device

### DISCUSSION THREAD

**Discussion by MrBLif** [rustdesk/rustdesk#9514] [General]
### Bug Description

Android 14 needs to start the service manually and allow to control the device. You can't set the allowance to control it. 

### How to Reproduce

Install it on android 14 by the website
Run it
Go to "share screen"
Try  to toggle the allowance to control

Also there you have to start the service

### Expected Behavior

That the service starts automatically
That I can control my device
That I will net get prompted daily if I allow the access

### Operating system(s) on local side and remote side

Android 14 (both) 

### RustDesk Version(s) on local side and remote side

1.3.1 - > 1.3.1

### Screenshots

![IMG_20240929_100254744~2](https://github.com/user-attachments/assets/68a0af82-efcd-4aa9-9c5d-28070d65fd35)
![IMG_20240929_100350928](https://github.com/user-attachments/assets/17f5931b-102c-470b-8d25-1ce05868bbf7)

**Comment by MrBLif** [comment]
ok for the first thing i have a workaround with macrodroid. But still i am not able to toggle the Touch Control :( Its still grey i can only see the screen but cannot control it :(

**Reply by MrBLif** [reply]
I split this into one two Problems. For this i have as is said a workaround ;)

**Reply by WachsenHaus** [reply]
How can I make the service start automatically even when the system reboots?

**Reply by xlionjuan** [reply]
not possible
